tests/test_fixity.py

```diff
@@ -0,0 +1,472 @@
+                        "files": {"missing": [], "changed": [], "untracked": []}
+                    },
+                    "timestamp": None,
+                },
+            },
+            spec=requests.Response,
+        ),
+    ],
+)
+def test_scanall(_get, capsys):
+    response = fixity.scanall(
+        ss_url=STORAGE_SERVICE_URL,
+        ss_user=STORAGE_SERVICE_USER,
+        ss_key=STORAGE_SERVICE_KEY,
+        session=SESSION,
+    )
+
+    assert response is True
+
+    captured = capsys.readouterr()
+    assert "Successfully scanned 2 AIPs" in captured.err
+
+
+@mock.patch(
+    "requests.get",
+    side_effect=[
+        mock.Mock(
+            **{
+                "status_code": 200,
+                "json.return_value": {
+                    "meta": {"next": None},
+                    "objects": [
+                        {
+                            "package_type": "AIP",
+                            "status": "UPLOADED",
+                            "uuid": str(uuid.uuid4()),
+                        },
+                        {
+                            "package_type": "AIP",
+                            "status": "UPLOADED",
+                            "uuid": str(uuid.uuid4()),
+                        },
+                    ],
+                },
+            },
+            spec=requests.Response,
+        ),
+        mock.Mock(
+            **{
+                "status_code": 401,
+                "json.return_value": {},
+            },
+            spec=requests.Response,
+            side_effect=Exception,
+        ),
+        mock.Mock(
+            **{
+                "status_code": 200,
+                "json.return_value": {
+                    "success": True,
+                    "message": "",
+                    "failures": {
+                        "files": {"missing": [], "changed": [], "untracked": []}
+                    },
+                    "timestamp": None,
+                },
+            },
+            spec=requests.Response,
+        ),
+        mock.Mock(
+            **{
+                "status_code": 401,
+                "json.return_value": {},
+            },
+            spec=requests.Response,
+            side_effect=Exception,
+        ),
+        mock.Mock(
+            **{
+                "status_code": 200,
+                "json.return_value": {
+                    "success": True,
+                    "message": "",
+                    "failures": {
+                        "files": {"missing": [], "changed": [], "untracked": []}
+                    },
+                    "timestamp": None,
+                },
+            },
+            spec=requests.Response,
+        ),
+    ],
+)
+def test_scanall_handle_exceptions(_get):
+    response = fixity.scanall(
+        ss_url=STORAGE_SERVICE_URL,
+        ss_user=STORAGE_SERVICE_USER,
+        ss_key=STORAGE_SERVICE_KEY,
+        session=SESSION,
+    )
+
+    assert response is False
```

Comment:
Should this contain additional assertions like errors messages being printed like in other exception scenarios above?